gitlab/v4/objects/projects.py

```diff
@@ -849,6 +851,49 @@ def import_project(
             "/projects/import", post_data=data, files=files, **kwargs
         )
 
+    def remote_import(
+        self,
+        url: str,
+        path: str,
+        name: Optional[str] = None,
+        namespace: Optional[str] = None,
+        overwrite: bool = False,
+        override_params: Optional[Dict[str, Any]] = None,
+        **kwargs: Any,
+    ) -> Union[Dict[str, Any], requests.Response]:
+        """Import a project from an archive file stored on a remote URL.
+
+        Args:
+            url: URL for the file containing the project data to import
+            path: Name and path for the new project
+            name: The name of the project to import. If not provided,
+                defaults to the path of the project.
+            namespace: The ID or path of the namespace that the project
+                will be imported to
+            overwrite: If True overwrite an existing project with the
+                same path
+            override_params: Set the specific settings for the project
+            **kwargs: Extra options to send to the server (e.g. sudo)
+
+        Raises:
+            GitlabAuthenticationError: If authentication is not correct
+            GitlabListError: If the server failed to perform the request
```

Comment:
This isn't exactly right, but I see we have the same in the other one. We'll fix that in all of htem later.